### PR TITLE
Make bipartite nodes compatible with learn2branch

### PR DESCRIPTION
**Pull request by dchetelat**
I doublechecked if the bipartite node features in Ecole matched the old learn2branch node features. it wasn't the case, but after the following fixes I now get the same results.

```
> np.abs(new_col_feat - col_feat['values']).max()
2.9450719307710926e-08
> np.abs(new_row_feat - row_feat['values'][new_to_old, :]).max()
7.080813868576286e-08
```

Ideally, we would also add a unit test to make them match; however, this would require the developer to have a custom version of PySCIPOpt I think? I don't really need how to make this light.

In addition, this is my first pull request. Is there tools I need to run on my PR?

### CODE REVIEW

File: libecole/src/scip/row.cpp [hunk 1]
```diff
@@ -33,7 +33,8 @@ int RowProxy::n_lp_nonz() const noexcept {
 }
 
 real RowProxy::l2_norm() const noexcept {
-	return SCIProwGetNorm(value);
+	auto norm = SCIProwGetNorm(value);
+	return norm > 0 ? norm : 1.;
```
Comment (AntoinePrv) [inline libecole/src/scip/row.cpp:37]: I do not feel this belongs here, as it does not match the name of the function.
Should go in `NodeBipartite`.
https://github.com/ds4dm/ecole/blob/3d6f3405223f93193b72b2acc278ea4f009a70a5/libecole/src/observation/nodebipartite.cpp#L35
Should go https://github.com/ds4dm/ecole/blob/3d6f3405223f93193b72b2acc278ea4f009a70a5/libecole/src/observation/nodebipartite.cpp#L84

Comment (dchetelat) [reply libecole/src/scip/row.cpp]: I changed the code according to your recommendations.

File: libecole/src/observation/nodebipartite.cpp [hunk 2]
```diff
@@ -85,12 +85,14 @@ static auto extract_row_feat(scip::Model const& model) {
 
 	auto extract_row = [n_lps, obj_l2_norm](auto& iter, auto const row, bool const lhs) {
 		value_type const sign = lhs ? -1. : 1.;
-		value_type const row_l2_norm = static_cast<value_type>(row.l2_norm());
+		value_type row_l2_norm = static_cast<value_type>(row.l2_norm());
+		if (row_l2_norm == 0) row_l2_norm = 1.;
```
Comment (AntoinePrv) [inline libecole/src/observation/nodebipartite.cpp:89]: Thanks. Should the change also be done for the columns features as well?
https://github.com/ds4dm/ecole/blob/170e7dab562164b234c96c8483c2f8d0fc3d14dd/libecole/src/observation/nodebipartite.cpp#L35

Comment (dchetelat) [reply libecole/src/observation/nodebipartite.cpp:89]: You mean setting it equal to 1 if it's equal to zero? The `get_obj_norm` function you coded already does that, no?

https://github.com/ds4dm/ecole/blob/170e7dab562164b234c96c8483c2f8d0fc3d14dd/libecole/src/observation/nodebipartite.cpp#L26-L29

Comment (AntoinePrv) [reply libecole/src/observation/nodebipartite.cpp:89]: Ha... Guess we need to make it similar to the other one then.

**Pull request merged**